babeld/babel_zebra.c

```diff
@@ -237,26 +237,26 @@ babel_zebra_connected (struct zclient *zclient)
 
 void babelz_zebra_init(void)
 {
-    zclient = zclient_new_notify(master, &zclient_options_default);
-    zclient_init(zclient, ZEBRA_ROUTE_BABEL, 0, &babeld_privs);
-
-    zclient->zebra_connected = babel_zebra_connected;
-    zclient->interface_add = babel_interface_add;
-    zclient->interface_delete = babel_interface_delete;
-    zclient->interface_up = babel_interface_up;
-    zclient->interface_down = babel_interface_down;
-    zclient->interface_address_add = babel_interface_address_add;
-    zclient->interface_address_delete = babel_interface_address_delete;
-    zclient->redistribute_route_add = babel_zebra_read_route;
-    zclient->redistribute_route_del = babel_zebra_read_route;
-
-    install_element(BABEL_NODE, &babel_redistribute_type_cmd);
-    install_element(ENABLE_NODE, &debug_babel_cmd);
-    install_element(ENABLE_NODE, &no_debug_babel_cmd);
-    install_element(CONFIG_NODE, &debug_babel_cmd);
-    install_element(CONFIG_NODE, &no_debug_babel_cmd);
-
-    install_element(VIEW_NODE, &show_debugging_babel_cmd);
+	zclient = zclient_new(master, &zclient_options_default);
+	zclient_init(zclient, ZEBRA_ROUTE_BABEL, 0, &babeld_privs);
```

Comment:
babeld uses [4 spaces for indentation](https://github.com/FRRouting/frr/blob/master/doc/developer/workflow.rst#L609-L613), we should preserve its different coding style here.

Reply:
fixed